### PR TITLE
166 Refactored settingsScreen, added sliders.

### LINKED ISSUE #166: Fix layout of settingsScene
- [x] Button layout correctly in the center
- [ ] Two sliders that indicate the volume for soundtrack/-effects
- [ ] The Toggle buttons disable the sliders when the toggle is off

### PR DESCRIPTION
**Pull request by MKerkhof**
Will close #166 

Please review the new layout of the settingscreen, now with volume sliders :)

<feature, user-interface

### CODE REVIEW

File: src/main/java/nl/tudelft/ti2206/group9/gui/scene/SettingsScene.java [hunk 1]
```diff
@@ -95,37 +105,85 @@
...
+        slider.setShowTickLabels(true);
+        slider.setShowTickMarks(true);
+        slider.setMaxWidth(width);
+        slider.setMax(MAX_VOLUME);
+        slider.setMin(0);
+        slider.setTooltip(new Tooltip("Adjust volume"));
+        slider.setValue(MAX_VOLUME / 2);
+        slider.setBlockIncrement(1);
+        slider.setMajorTickUnit(ticks);
+        slider.setMinorTickCount(ticks);
```
Comment (mpsijm) [inline src/main/java/nl/tudelft/ti2206/group9/gui/scene/SettingsScene.java]: I think this the evil: minorTickCount should be 4.

<review, logic

**Comment by mpsijm** [comment]
One comment about the styling: [https://gyazo.com/82400da04e5b94d3ecb99bb2112fbb27](https://gyazo.com/82400da04e5b94d3ecb99bb2112fbb27)
It looks like the minorTicks are now at `0.83`, `1.67`, `2.5`, ..., while I'd expected them to be integers.

Maybe you can let the EndToEnd test also test the sliders, or is that easier to do when the back-end gets connected? Then it's also possible to check if the volume has really changed internally :)

I have fixed one last PMD warning for you; the Checkstyle warning at `HighScoreScene:112` (space after comma) should be resolved in master already (I hope :innocent:)

Overall, well done! When you fix my comments, it's ready to be merged! :)

<review, user-interface

**Comment by MathiasMeuleman** [comment]
- [ ] My sliders show display marks at 0, 4, 8 and 10
 - [ ] The down half of my Back-button has disappeard from the screen

Furthermore the code looks nice now

<review

**Comment by thervh70** [comment]
My version shows the same graphical bugs as @MathiasMeuleman
I really like the functionality you've introduced, with the disabling of the sliders.

< review

**Comment by MKerkhof** [comment]
@thervh70 @MathiasMeuleman @mpsijm Please look at it again, I have processed your feedback and made some changes :)

<review, user-interface

**Comment by mpsijm** [comment]
Thanks for the fix, nicely done! Ready to be merged :)

<review, user-interface

**Comment by thervh70** [comment]
Yes, this looks good, ready to be merged!

< review

**Pull request merged**